website/util/metrics.py

```diff
@@ -28,6 +28,14 @@ def campaign_source_tag(campaign_name):
     return f'source:campaign|{campaign_name}'
 
 
+def unregistered_created_source_tag(reffer_id):
+    return f'source:unregistered_created|{reffer_id}'
+
+
+def institution_source_tag(institution_id):
+    return f'source:institution|{institution_id}'
```

Comment:
Probably a @brianjgeiger or maybe product question, should tag for institution called `source:campaign|{campaign_name}` where `campaign_name` is something like "osf_institutions".